app/models/manageiq/providers/block_storage_manager.rb

```diff
@@ -0,0 +1,7 @@
+class ManageIQ::Providers::BlockStorageManager < ManageIQ::Providers::StorageManager
```

Comment:
This should be a mixin module instead of a class - something like: `BlockStorageManagerMixin`.